src/main/java/org/fundacionjala/coding/jorge/DnaStrand.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public final class DnaStrand {
+    /**
+     *
+     */
+    private DnaStrand() {
+    }
+
+    /**
+     * @param dna given type String
+     * @return cadDna given type String
+     * @serialData cadDna type String
+     * @serialData arrayDna
+     */
+    public static String makeComplement(final String dna) {
+        String cadDna = dna.toLowerCase();
+        char[] arrayDna = cadDna.toCharArray();
+        char[] arrayC = new char[cadDna.length()];
+        int i = 0;
+        for (char aChar : arrayDna) {
+            switch (aChar) {
```

Comment:
For this scenario don't use a switch case.
Consider using a Map or you can take advantage of String methods. (replaceAll)